test/test_memory.jl

```diff
@@ -36,7 +36,7 @@ zetas = [copy(tovoigt(mat.variables.zeta; offdiagscale=2.0))]
 
 n_cycles = 30
 ppc = 40
-t = range(0, n_cycles; length=n_cycles*ppc+1)
```

Comment:
Another way to do this would be
```
dt = inv(ppc)
t = 0:dt:n_cycles
```

After all, range is just returning `UnitRange` which can be described with `t0:dt:t1` syntax.